src/main/resources/org/jenkinsci/plugins/durabletask/executors/OnceRetentionStrategy/config.groovy

```diff
@@ -0,0 +1,7 @@
+package org.jenkinsci.plugins.durabletask.executors.OnceRetentionStrategy
+
+def f = namespace(lib.FormTagLib);
+
+f.entry(title: "Idle timeout", field: "idleMinutes") {
+    f.number(default: 0)
```

Comment:
Either `0` or `1` seems fine as a default.

> as soon as strategy runs it kills slave

This should only happen if there is no task running on it, obviously, so if you see differently then something else is wrong somewhere, but arbitrarily adjusting the idle timeout is not the fix. Any nonnegative value is legitimate here.